[2023-10-17T11:19:35.603+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-17T11:19:35.609+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-17T11:19:35.610+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:35.620+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-13 00:00:00+00:00
[2023-10-17T11:19:35.625+0000] {standard_task_runner.py:57} INFO - Started process 698 to run task
[2023-10-17T11:19:35.627+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-13T00:00:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpitlg_fhc']
[2023-10-17T11:19:35.628+0000] {standard_task_runner.py:85} INFO - Job 137: Subtask waiting_for_data
[2023-10-17T11:19:35.666+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:35.734+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-13T00:00:00+00:00'
[2023-10-17T11:19:35.740+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:35.740+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:35.741+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:35.741+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:35.748+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210513T000000, start_date=20231017T111935, end_date=20231017T111935
[2023-10-17T11:19:35.799+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:35.816+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:10.326+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-18T10:43:10.333+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-18T10:43:10.333+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:10.343+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-13 00:00:00+00:00
[2023-10-18T10:43:10.348+0000] {standard_task_runner.py:57} INFO - Started process 1661 to run task
[2023-10-18T10:43:10.350+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-13T00:00:00+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpjtbllzpn']
[2023-10-18T10:43:10.351+0000] {standard_task_runner.py:85} INFO - Job 136: Subtask waiting_for_data
[2023-10-18T10:43:10.391+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:10.450+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-13T00:00:00+00:00'
[2023-10-18T10:43:10.464+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:10.467+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210513T000000, start_date=20231018T104310, end_date=20231018T104310
[2023-10-18T10:43:10.475+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 136 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1661)
[2023-10-18T10:43:10.482+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:10.502+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:30.823+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-18T11:43:30.837+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-18T11:43:30.837+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:30.854+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-13 00:00:00+00:00
[2023-10-18T11:43:30.865+0000] {standard_task_runner.py:57} INFO - Started process 984 to run task
[2023-10-18T11:43:30.869+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-13T00:00:00+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp2qp4uwmp']
[2023-10-18T11:43:30.870+0000] {standard_task_runner.py:85} INFO - Job 225: Subtask waiting_for_data
[2023-10-18T11:43:30.937+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:31.053+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-13T00:00:00+00:00'
[2023-10-18T11:43:31.062+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:31.065+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:31.066+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:31.068+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:31.078+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210513T000000, start_date=20231018T114330, end_date=20231018T114331
[2023-10-18T11:43:31.132+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:31.193+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:44:26.553+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-19T12:44:26.575+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [queued]>
[2023-10-19T12:44:26.576+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:44:26.595+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-13 00:00:00+00:00
[2023-10-19T12:44:26.603+0000] {standard_task_runner.py:57} INFO - Started process 1197 to run task
[2023-10-19T12:44:26.607+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-13T00:00:00+00:00', '--job-id', '261', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpw_vetgks']
[2023-10-19T12:44:26.608+0000] {standard_task_runner.py:85} INFO - Job 261: Subtask waiting_for_data
[2023-10-19T12:44:26.685+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-13T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:44:26.794+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-13T00:00:00+00:00'
[2023-10-19T12:44:26.806+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:44:26.806+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:44:26.806+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:44:26.807+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:44:26.819+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210513T000000, start_date=20231019T124426, end_date=20231019T124426
[2023-10-19T12:44:26.861+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:44:26.920+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
